extensions/wikia/SpitfiresContributionExperiments/scripts/experiments-tracker.js

```diff
@@ -50,8 +54,15 @@ define('ext.wikia.spitfires.experiments.tracker', [
 		}
 	}
 
+	function prepareStructuredLabel(experiment, label) {
+		var group = abTest.getGroup(experiment),
+			userStatusLabel = getUserStatus(),
```

Comment:
A semi-colon `;` instead of a comma `,` and a new line after vars declaration for better readability